test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -45,6 +46,17 @@ TEST(QuicPlatformTest, QuicArraysize) {
 
 enum class TestEnum { ZERO = 0, ONE, TWO, COUNT };
 
+TEST(QuicPlatformTest, QuicBugTracker) {
+  if (false) {
```

Comment:
Instead of doing this, can you give a try to use `'EXPECT_DEATH'

Reply:
Done. PTAL.